ENIGMAsystem/SHELL/Platforms/Cocoa/CocoaMain.cpp

```diff
@@ -68,7 +86,7 @@ namespace enigma_user {
     usleep((ms % 1000) *1000);
   };
   
-  int parameter_count(){
+  int parameter_count() {
```

Comment:
this line

Reply:
That's a whitespace change you goofball, it's not like he changed the logic.

Reply:
dont change whitespace 4 no reason

Reply:
Sure... but it's one line... I mean.

Reply:
@RemoveRusky there is a reason. Consistency. Most of those have a space in between. Whovever wrote the code originally must've typo'd. Thus I corrected it.. Don't talk to me again until you can write a paragraph explaining why consistency and applying OCD to your code is so important.